metricbeat/module/kafka/consumergroup/consumergroup.go

```diff
@@ -0,0 +1,100 @@
+package consumergroup
+
+import (
+	"crypto/tls"
+
+	"github.com/Shopify/sarama"
+	"github.com/elastic/beats/libbeat/common"
+	"github.com/elastic/beats/libbeat/logp"
+	"github.com/elastic/beats/libbeat/outputs"
+	"github.com/elastic/beats/metricbeat/mb"
+	"github.com/elastic/beats/metricbeat/module/kafka"
+)
+
+// init registers the MetricSet with the central registry.
+func init() {
+	if err := mb.Registry.AddMetricSet("kafka", "consumergroup", New); err != nil {
+		panic(err)
+	}
+}
+
+// MetricSet type defines all fields of the MetricSet
+type MetricSet struct {
+	mb.BaseMetricSet
+
+	broker *kafka.Broker
+	topics nameSet
+	groups nameSet
+}
+
+type groupAssignment struct {
+	clientID   string
+	memberID   string
+	clientHost string
+}
+
+var debugf = logp.MakeDebug("kafka")
```

Comment:
partitions is using `kafka` as well. we should clean this in a followup PR. Do we follow a debug selector naming standard in metricbeat?